index.html

```diff
@@ -4,9 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Hosting Stron - Nieograniczony Hosting i Darmowa Domena -</title>
+    <link rel="stylesheet" href="./css/main.css">
 </head>
+
+
+
 <body>
-    
+<header class="support">
```

Comment:
Ten header nie jest wewnatrz zadnego semantycznego znacznika (`main`, `section`, `aside`, `article`), wiec google bedzie myslec ze to jest glowny header strony, a to nie jest prawda, bo glowny naglowek jest nizej na stronie